src/terraform/index_test.go

```diff
@@ -69,7 +69,7 @@ provider "aws" {
 }
 
 module "aws" {
-  source = "./aws"
+  source = "git@github.com:Originate/exosphere.git//src//terraform//modules//aws?ref=8786f912"
```

Comment:
Can we put the commit in a variable so if we want to update it we only need to do it in one place?